less/home-page.less

```diff
@@ -52,10 +60,30 @@
     border-top: 2px dashed @navigation-border;
     display: flex;
     justify-content: space-between;
+    @media @mobile {
+      flex-direction: column;
+      width: 100%;
```

Comment:
check the class name. This one tricked me up too. I was trying to zoom through and missed this. Felt dumb about it when my PM called me out but it was definitely a good lesson to learn.